### ISSUE TITLE
Proportional resizing of image in card

### ISSUE THREAD

**Issue by Saentist** [t1gr0u/rain-gauge-card#7]
Currently if size of card is 1x1 not standard 1x2 text is over image.
![image](https://user-images.githubusercontent.com/16975883/236419596-7dd61596-5797-4e0c-8c91-eed6a8424318.png)

``` yaml
type: horizontal-stack
cards:
  - type: custom:rain-gauge-card
    entity: sensor.daily_rain_rate_2
    hourly_rate_entity: sensor.hourly_rain_rate_2
  - type: custom:uv-index-card
    entity: sensor.uv_index
```

Gap bellow UV index is also strange.

**Comment by Saentist** [comment]
Upper image from 2K screen
If screen size is 1366x768
![image](https://user-images.githubusercontent.com/16975883/236702525-e2dfff63-51a8-431d-9452-15833a0c36b2.png)

On phone even worst.

**Comment by t1gr0u** [comment]
Thank you for letting me know 👍

**Comment by t1gr0u** [comment]
This is now fixed in v1.1.0
